aptos-move/framework/src/natives/permissioned_signer.rs

```diff
@@ -95,8 +95,8 @@ pub fn make_all(
 ) -> impl Iterator<Item = (String, NativeFunction)> + '_ {
     let natives = [
         (
-            "is_permissioned_signer",
-            native_is_permissioned_signer as RawSafeNative,
+            "is_permissioned_signer_impl",
+            native_is_permissioned_signer_impl as RawSafeNative,
```

Comment:
you should put both in the list here (pointing to the same `native_is_permissioned_signer_impl`, so you can deprecate `is_permissioned_signer_impl` later